### DISCUSSION TITLE
Welcome to NaSC Discussions!

### DISCUSSION THREAD

**Discussion by parnoldx** [parnoldx/nasc#163] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by hasrack** [comment]
Is there an documentation somewhere? is it possible to know time of a place with this? Something like "Times in Berlin" or is it only for multiplication?

**Reply by parnoldx** [reply]
It has an integrated "help" but more like a reference of all functions.
Not right now. Thought of this in the past but never done this.
